src/train-classifier/src/test/scala/VerifyTrainClassifier.scala

```diff
@@ -96,60 +94,16 @@ class VerifyTrainClassifier extends Benchmarks with EstimatorFuzzing[TrainClassi
       (1, 2, 0.40, 0.50, bird, dog),
       (0, 3, 0.78, 0.99, dog, bird),
       (1, 4, 0.12, 0.34, cat, dog)))
-      .toDF(mockLabelColumn, "col1", "col2", "col3", "col4", "col5")
+      .toDF(mockLabelCol, "col1", "col2", "col3", "col4", "col5")
 
     val model1 = new ValueIndexer().setInputCol("col4").setOutputCol("col4").fit(dataset)
     val model2 = new ValueIndexer().setInputCol("col5").setOutputCol("col5").fit(dataset)
     val catDataset = model1.transform(model2.transform(dataset))
 
-    val logisticRegressor = TrainClassifierTestUtilities.createLogisticRegressor(mockLabelColumn)
-    TrainClassifierTestUtilities.trainScoreDataset(mockLabelColumn, catDataset, logisticRegressor)
-
-    val randomForestClassifier = TrainClassifierTestUtilities.createRandomForestClassifier(mockLabelColumn)
-    TrainClassifierTestUtilities.trainScoreDataset(mockLabelColumn, catDataset, randomForestClassifier)
-  }
-
-  test("Verify saving and loading a trained classifier model") {
-    val dataset: DataFrame = createMockDataset
```

Comment:
I would recommend keeping this, even though it is checked in the fuzzing, this also validates saving twice and some properties and it is good to have just in case fuzzing tests "fail to fail" since they are automated

Reply:
(almost too automated)

Reply:
THis test does some very sketchy things with the working directory that makes it very brittle.  We should add these sorts of checks in the fuzzing infrastructure so everyone can benefit.